sycl/doc/UsersManual.md

```diff
@@ -45,8 +45,8 @@ and not recommended to use in production environment.
     currently overrides all the other specified SYCL targets when enabled.)
 
     Special target values specific to Intel, NVIDIA and AMD Processor Graphics
-    support are accepted, providing a streamlined interface for AOT. Only one of
-    these values at a time is supported.
+    support are accepted, providing a streamlined interface for AOT.
+    A comma-separated list of valid Intel,NVidia and AMD Processor Graphics values are supported.
```

Comment:
```suggestion
    A comma-separated list of valid Intel, NVIDIA and AMD Processor Graphics
    values are supported.
```
I don't know what the preferred way of representing `NVIDIA` is, but we should be consistent with usage (here and line 47)